src/HealthChecks.Publisher.Seq/SeqPublisher.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+
+namespace HealthChecks.Publisher.Seq
+{
+    public class SeqPublisher : IHealthCheckPublisher, IDisposable
+    {
+        private const string EVENT_NAME = "AspNetCoreHealthCheck";
+        private const string METRIC_STATUS_NAME = "AspNetCoreHealthCheckStatus";
+        private const string METRIC_DURATION_NAME = "AspNetCoreHealthCheckDuration";
+
+        private readonly SeqOptions _options;
+        private readonly HttpClient _httpClient;
+
+        public SeqPublisher(SeqOptions options)
+        {
+            _options = options;
+            _httpClient = new HttpClient
```

Comment:
We are changing the code to use IHttpClientFactory, can you check the Uri HealthCheck or IdSvr in order  to check this?